test/powershell/Language/Classes/Scripting.Classes.Modules.Tests.ps1

```diff
@@ -71,3 +71,41 @@ Import-Module Random
     }
 
 }
+
+Describe 'Module reloading with Class definition' -Tags "CI" {
+
+    BeforeAll {
+        Set-Content -Path TestDrive:\TestModule.psm1 -Value @'
+$passedArgs = $args
+class Root { $passedIn = $passedArgs }
+function Get-PassedArgsRoot { [Root]::new().passedIn }
+function Get-PassedArgsNoRoot { $passedArgs }
+'@
+    }
+
+    AfterEach {
+        Remove-Module TestModule -Force -ErrorAction SilentlyContinue
+    }
+
+    It "Class execution reflects changes in module reloading with '-Force'" {
+        Import-Module TestDrive:\TestModule.psm1 -ArgumentList 'value-1'

```

Comment:
I even thought about repeating code in the tests, but it seems `testcase` here is not applicable.